simple_use_case/kadistudio/README.md

```diff
@@ -0,0 +1,33 @@
+# KadiStudio
+Implementation of the simple use case with KadiStudio. `KadiStudio` is a software component of the 
+Kadi4Mat research data infrastructure, that is dedicated to model, visualise and execute scientific workflows using a GUI. 
+
+## Installation
+`KadiStudio` can be installed using the debian package provided [here](https://kadi4mat.iam-cms.kit.edu/records/6479).
+To execute a workflow in KadiStudio, the two software components `process-manager` and `process-engine` are
+required. They are available as debian packages and can be installed using the following commands:
+```sh
+wget https://gitlab.com/iam-cms/workflows/process-manager/-/jobs/artifacts/master/download?job=pack_deb -O process_manager.zip
+wget https://gitlab.com/iam-cms/workflows/process-engine/-/jobs/1764662488/artifacts/download -O process_engine.zip
+unzip process_*.zip
+apt-get install --yes ./build/*.deb
+```
+The tools used within KadiStudio have to be installed separately using the python library
+`workflow-nodes`, available with:
+
+```sh
+pip install workflow-nodes
+```
```

Comment:
That's right, as KadiStudio creates a visual representation of each used tool, we need to define a machine readable description of each tool that we are using. This machine readable description is provided by the --xmlhelp option that can be added to existing tools using the [xmlhelpy](https://pypi.org/project/xmlhelpy/) library inside of a wrapper script.
Tools which have already been equipped with this xmlhelp option through wrapper scripts are collected inside the `workflow-nodes` library.  So in short the workflow-nodes are just wrapper scripts of existing tools such as octave, plot_matplotlib etc. that enable their visual representation in KadiStudio.
As we needed some specific tools such as gmsh, tectonic etc. I added new wrapper scripts to the workflow-nodes library. These are, however, not published yet, which is why I am not using `pip install workflow-nodes` inside the install_components.sh script, but install my latest workflow-nodes branch directly.

Reply:
Okay, thanks for the clarification. So basically every user of KadiStudio will have to write those wrappers to register any (version of a) tool he/she would like to use. It would be nice to have a short explanation (or link to the documentation of `workflow-nodes` where this is explained) of what is necessary to use any tool with KadiStudio. Judging from the wrapper for [meshio](https://gitlab.com/lars153/workflow-nodes/-/blob/e1e0f20beadaf0ff13e7fbfec2bbd3c6cd141781/workflow_nodes/misc/meshio_convert.py) this is quite easy.